features/support/run_helpers.rb

```diff
@@ -30,6 +30,13 @@ def git_town_command? command
 end
 
 
+# Removes the given output from all formatting,
+# to make it comparable with other outputs
+def normalize_output out
+  out.gsub(/\S*/, ' ').strip
+end
```

Comment:
If you are trying to remove formatting as the comment says. Check out the function [unformatted_last_run_output](https://github.com/Originate/git-town/blob/master/features/support/run_helpers.rb#L106)